components/infobox/wikis/starcraft/infobox_series_custom.lua

```diff
@@ -0,0 +1,205 @@
+---
+-- @Liquipedia
+-- wiki=starcraft
+-- page=Module:Infobox/Series/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Namespace = require('Module:Namespace')
+local Series = require('Module:Infobox/Series')
+local SeriesTotalPrize = require('Module:SeriesTotalPrize')
+local Tier = require('Module:Tier')
+local Variables = require('Module:Variables')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Builder = require('Module:Infobox/Widget/Builder')
+local Class = require('Module:Class')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local _TIER_MODE_TYPES = 'types'
+local _TIER_MODE_TIERS = 'tiers'
+local _INVALID_TIER_WARNING = '${tierString} is not a known Liquipedia '
+	.. '${tierMode}[[Category:Pages with invalid ${tierMode}]]'
+
+local CustomInjector = Class.new(Injector)
+
+local CustomSeries = {}
+
+local _args
+
+function CustomSeries.run(frame)
+	local series = Series(frame)
+	_args = series.args
+
+	series.createWidgetInjector = CustomSeries.createWidgetInjector
+
+	return series:createInfobox(frame)
+end
+
+function CustomSeries:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{name = 'Patch', content = {CustomSeries._getPatch()}})
```

Comment:
Not really sold that this is a field that fits series field?

Reply:
well if a series spreads over several patches the statistics about that series have a different value compared to it only being on one patch

because the patch changes actually might have an influence on the statistics

hence i think displaying the patch data is a good idea for series

Reply:
Ok